src/controllers/farcaster.ts

```diff
@@ -4,6 +4,20 @@ import { PostModel } from '@/models/Post'
 import { notFound } from '@hapi/boom'
 import fetch from 'node-fetch'
 
+type Cast = {
+  merkleRoot: string
+  threadMerkleRoot: string
+  body: {
+    type: string
```

Comment:
Maybe we should use `type` here to avoid direct comparison like you do on line 60?